src/view/reviewManager.ts

```diff
@@ -110,7 +128,7 @@ export class ReviewManager implements vscode.DecorationProvider {
 			}
 		}));
 
-		this._disposables.push(_repository.state.onDidChange(e => {
+		this._disposables.push(this._repository.state.onDidChange(e => {
```

Comment:
We may want to move this one to something like `registerListeners` as others are commands but this one is not.